### ISSUE TITLE
Is there an Android demo in this library

### ISSUE THREAD

**Issue by huangfei1love** [nihui/realsr-ncnn-vulkan#36]
I want to use this library on Android machines, but not in the termux way, but in the native app way。

**Comment by ArchieMeng** [comment]
No, there is no android demo based on RealSR ncnn Vulkan yet. However, there are some demos based on other models. They are mentioned in the [awesome-ncnn](https://github.com/zchrissirhcz/awesome-ncnn).

**Comment by huangfei1love** [comment]
> No, there is no android demo based on RealSR ncnn Vulkan yet. However, there are some demos based on other models. They are mentioned in the [awesome-ncnn](https://github.com/zchrissirhcz/awesome-ncnn).
is this model only supports 4x scale?Can I use it in Android applications.

**Comment by tumuyan** [comment]
@nihui  
I noticed that you have made styletransfer-ncnn for Android, and realesrgan-ncnn is based on the realsr-ncnn. Can you please make an Android version of realsr-ncnn? It may be easy for you, and very useful for many people. I have tried, but I did not succeed  cause of not professional enough.
Thanks.
